docs/source/blog/neuroblueprint.md

```diff
@@ -0,0 +1,182 @@
+---
+blogpost: true
+date: 25 April 2024
+author: Joe Ziminski, Niko Sirmpilatze
+location: London, UK
+category: Blog
+language: English
+image: 1
+---
+
+# Data organisation with NeuroBlueprint
+*The challenge of unstandardised data in systems neuroscience*
+
+**estimated reading time: 10 minutes**
+
+```{image} /_static/blog_images/neuroblueprint/errant-science-data-organisation.jpg
+:align: center
+:width: 75%
+```
+
+The increasing use of software to automate acquisition and 
+analysis pipelines should be revolutionising how researchers collaborate.
+It should now be commonplace to quickly and conveniently plug newly acquired 
+data into community pipelines for common preprocessing and analysis steps.
+
+Unfortunately this is typically not the case, with researchers 
+often re-coding pipelines from scratch and in isolation.
+
+A lack of standardised data format for systems neuroscience
+is a major obstacle to realising the benefits of automation.
+Have you ever borrowed a colleagues analysis scripts, only to spend
+the next few hours changing the code structure to fit your data?
+Lack of data standardisation wastes time, prohibits effective 
+collaboration and at worst leads to mistakes in analysis and reporting.
+It becomes impossible for the community to build high-level preprocessing 
+and analysis pipelines.
+
+To address these problems, the [Neuroinformatics Unit](https://neuroinformatics.dev/) at the SWC have 
+developed an easy-to-use data standardisation framework, 
+[**NeuroBlueprint**](https://neuroblueprint.neuroinformatics.dev/). This blog motivates the need for
+data standardisation at the SWC and provides an introduction
+to **NeuroBlueprint**,
+highlighting its
+place within the current data standardisation landscape.
+
+In a companion posts, we introduce [**datashuttle**](https://datashuttle.neuroinformatics.dev), a tool to 
```

Comment:
Are we releasing these two posts at the same time? I feel we'll only get someone's attention once, so they should have immediate access to the follow on reading if they're interested.

Reply:
I think in this particular case, I'll opt for releasing them together, with them being cross-linked.

Reply:
Agreed, I think best approach is to:

1) get both posts done on their respective PRs
2) merge both
3) quick PR to link the pages together via sphinx cross-referencing

Reply:
They can be merged whenever really, nobody will read them until we advertise anyway.